### PR TITLE
Deprecate confusing option names

### PR DESCRIPTION
**Pull request by Hypnosphi**
"Left Panel" => "Stories Panel"
"Down Panel" => "Addon Panel"

Old option names stay fully supported for now, but produce a deprecation warning in development

I'll update the addon-options docs as soon as it's published

# How to test:
- Roll back the changes in `cra-kitchen-sink` and run it
- Remove `setOptions` from `cra-kitchen-sink` (because of #1690) and add `left=0&down=0` to URL

### CODE REVIEW

**Review by igor-dv — APPROVED** [review]
LGTM. (Let's not merge until #1693, please)

File: lib/ui/src/libs/key_events.js [hunk 1]
```diff
@@ -2,14 +2,14 @@ import keycode from 'keycode';
 
 export const features = {
   FULLSCREEN: 1,
-  DOWN_PANEL: 2,
-  LEFT_PANEL: 3,
+  ADDON_PANEL: 2,
+  NAVIGATION_PANEL: 3,
```
Comment (UsulPro) [inline lib/ui/src/libs/key_events.js]: Why everywhere it's called "Stories Panel" but here is "Navigation Panel"?

**Comment by UsulPro** [comment]
Great! :+1: 
This change is something that we really need to have done!

But I prefer to have a strong "How to test" section and maybe better to merge it right before the next minor releasing

**Comment by Hypnosphi** [comment]
> a strong "How to test" section

I've made one, but both cases are covered by added tests

File: lib/ui/src/libs/key_events.js [hunk 2]
```diff
@@ -2,14 +2,14 @@ import keycode from 'keycode';
 
 export const features = {
   FULLSCREEN: 1,
```
Comment (UsulPro) [inline lib/ui/src/libs/key_events.js]: Maybe it's a good time to also switch from numbers to literals here, what do you think?

File: lib/ui/src/libs/key_events.js (hunk 1 above)
Comment (Hypnosphi) [inline lib/ui/src/libs/key_events.js]: My bad, skipped when refactoring

File: lib/ui/src/libs/key_events.js (hunk 2 above)
Comment (Hypnosphi) [inline lib/ui/src/libs/key_events.js]: Nice idea

**Review by danielduan — CHANGES_REQUESTED** [review]
Don't forget to update the readme file too. The docs might need an update as well.

https://github.com/storybooks/storybook/blob/master/addons/options/README.md

**Comment by Hypnosphi** [comment]
> I'll update the addon-options docs as soon as it's published

**Comment by Hypnosphi** [comment]
@danielduan I've changed base fork to `release/3.3`, this allowed to update README right now

**Review by danielduan — APPROVED** [review]

**Pull request merged**